nbs/models.hint.ipynb

```diff
@@ -17,18 +17,27 @@
    ]
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #24.    S = np.array([[1., 1., 1., 1.],
</pre><p>If you include a screen shot of the slide where we introduce Hierachical Notation, this matrix (and the whole) example becomes a lot clearer.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/557/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Thanks. Added the hierarchical plots.